TS2310: Type 'Context' recursively references itself as a base type.

Type:
- [x] Bug
- [ ] Feature
- [ ] Help   

Version:   
0.3.0

Description:  
```
node_modules/koa-custom-response/typings/response.d.ts:5:13 - error TS2310: Type 'Context' recursively references itself as a base type.

5   interface Context extends Koa.Context {
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "koa-custom-response",
-  "version": "0.3.0",
+  "version": "0.3.1",
   "description": "standard and convenient KOA2 custom responses.",
   "main": "./dist/index.js",
   "module": "./dist/index.js",
@@ -8,7 +8,7 @@
   "scripts": {
     "test": "nyc mocha",
     "cover:local": "nyc mocha; open coverage/lcov-report/index.html",
-    "cover:hook": "nyc mocha; cat ./coverage/lcov.info | coveralls; exit",
+    "cover:hook": "npm run clean:cover && nyc mocha | coveralls",
     "commit:hook": "commitlint --from origin/master --to HEAD",
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "start": "export NODE_ENV=development; ts-node examples/app.ts",
@@ -53,8 +53,7 @@
       "./node_modules/ts-node"
     ],
     "reporter": [
-      "lcov",
-      "text",
+      "text-lcov",
       "text-summary"
     ],
     "cache": false,

--- a/test/utils.nyc.ts
+++ b/test/utils.nyc.ts
@@ -13,7 +13,7 @@ const closeServer = (app: Server): void => {
   timer = global.setTimeout(() => {
     app.close()
     clearTimeout(timer)
-  }, 100)
+  }, 5000)
 }
 
 export const app = async(): Promise<ChaiHttp.Agent> => {

--- a/typings/response.d.ts
+++ b/typings/response.d.ts
@@ -2,7 +2,7 @@ import * as Koa from 'koa'
 
 declare module 'koa' {
   
-  interface Context extends Koa.Context {
+  interface Context {
     ok: (data?: object | string) => void
     created: (data?: object | string) => void
     noContent: () => void
@@ -18,4 +18,4 @@ declare module 'koa' {
 
 declare function KoaCustomResponse(): Koa.Middleware
 declare namespace KoaCustomResponse { }
-export = KoaCustomResponse
+export { KoaCustomResponse }

